src/compiler/types.ts

```diff
@@ -128,6 +128,7 @@ module ts {
         PrivateKeyword,
         ProtectedKeyword,
         PublicKeyword,
+        AbstractKeyword,

```

Comment:
Jup will do, didn't wrote a (full) test-suite yet because I wanted to know if the code was a bit in the right places